### PR TITLE
Common repo

### PR DESCRIPTION
**Pull request by rubene**
🐘 
- Move this role into its own repo because is a dependency on web_server
  role (now ansible-nginx) which is also moving to a separate repo.

### CODE REVIEW

**Comment by jessedoyle** [comment]
:elephant::elephant::elephant:

File: templates/logrotate.rails [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/srv/**/log/*.log {
+  su root root
+  daily
+  size 50M
```
Comment (jbabe) [inline templates/logrotate.rails:4]: i'm guessing we are leaving it up to users to decide if they need variables? i.e if I have a template that expects

``` jinja
 size {{ logrotate_max_size }} 
```

I'd have to define `logrotate_max_size` in my playbook?

**Comment by jbabe** [comment]
🐘🐘🐘

File: templates/logrotate.rails (hunk 1 above)
Comment (jessedoyle) [inline templates/logrotate.rails:4]: @jbabe - I believe that's the way @rubene and I envisioned it.

**Pull request merged**